src/Sinch/Numbers/Active.cs

```diff
@@ -91,7 +101,9 @@ public ActiveNumbers(string projectId, Uri baseAddress,
             _http = http;
         }
 
-        public Task<ListActiveNumbersResponse> List(ListActiveNumbersRequest request, CancellationToken cancellationToken = default)
+
+        public Task<ListActiveNumbersResponse> List(ListActiveNumbersRequest request,
```

Comment:
I'm curious about why the formatting changed as there is no configuration change in this PR. Is it linked to the IDE? I just want to make sure this is intentional and won't be reverted by an auto-formatting in another PR.

Reply:
Auto formatted by IDE agnostic `dotnet format`